components/git/release.js

```diff
@@ -1,14 +1,24 @@
+import auth from '../../lib/auth.js';
 import CLI from '../../lib/cli.js';
 import ReleasePreparation from '../../lib/prepare_release.js';
+import ReleasePromotion from '../../lib/promote_release.js';
+import TeamInfo from '../../lib/team_info.js';
+import Request from '../../lib/request.js';
 import { runPromise } from '../../lib/run.js';
 
-export const command = 'release [newVersion|options]';
+export const command = 'release [prid|options]';
 export const describe = 'Manage an in-progress release or start a new one.';
 
 const PREPARE = 'prepare';
 const PROMOTE = 'promote';
+const RELEASERS = 'releasers';
 
 const releaseOptions = {
+  dryRun: {
```

Comment:
It should be enabled by default

Reply:
I'm fine with having `--dry-run` being the default, I agree it somewhat makes sense; does anyone have a suggestion for a good name for such option? I assume `--no-dry-run` is not very self explanatory.

Reply:
`--run` ?

Reply:
Went with `--run` in https://github.com/nodejs/node-core-utils/pull/835/commits/5ccb47472540e82d2da38fa79fd750ae9804b92d